docs/faq.md

```diff
@@ -38,3 +38,43 @@ Project Mu makes liberal use of multiple repositories due to the mixture of requ
 repos are split for technical reasons, some for organizational, and some for legal.
 
 For details, see "Repo Philosophy" in [What and Why](WhatAndWhy/overview.md#repo-philosophy).
+
+## Is there any stronger guidance than "Less is More"?
+
+Since Mu is derived from Edk2, the Edk2 coding standards are a good foundation, as is the existing code in the 
+TianoCore repositories. Beyond that, here are a few other musings:
+
+#### Forgotten lessons in firmware
+Firmware, no matter what the flavor of it, all has the same end goal: to securely boot some sort of payload. Features

```

Comment:
Not putting an empty line after headings will usually cause a markdownlint error. I see there's one after the previous heading `## Is there any stronger guidance than "Less is More"?`.

Did you check if it's needed here?